docs/content/config/advanced/full-text-search.md

```diff
@@ -103,6 +103,60 @@ While indexing is memory intensive, you can configure the plugin to limit the am
     ```
     docker-compose exec mailserver doveadm fts optimize -A
     ```
+    Or as well as with [Spamassassin example][docs-faq-sa-learn-cron] you can use the internal instance of `cron` within `docker-mailserver` to avoid possible errors if the mail-server is not running:
```

Comment:
```suggestion
    Or like the [Spamassassin example][docs-faq-sa-learn-cron] shows, you can instead use `cron` from within `docker-mailserver` to avoid potential errors if the mail-server is not running:
```